amgut/db/0042.sql

```diff
@@ -0,0 +1,129 @@
+CREATE TABLE ag."source" (
+	source_id            uuid DEFAULT ag.uuid_generate_v4() ,
+	source_name          varchar(100)  ,
```

Comment:
Is the 100 chars limitation implemented in the GUI? If not either implement there or remove here.